packages/app/src/cli/models/app/loader.ts

```diff
@@ -65,16 +65,18 @@ class AppLoader {
     this.appDirectory = await this.findAppDirectory()
     const configurationPath = await this.getConfigurationPath()
     const configuration = await this.parseConfigurationFile(AppConfigurationSchema, configurationPath)
-    const extensionsPath = path.join(this.appDirectory, `${blocks.extensions.directoryName}`)
+    const extensionDirectories = configuration.extensionDirectories.map((extensionDirectory) => {
+      return path.join(this.appDirectory, extensionDirectory)
+    })
     const dotenv = await this.loadDotEnv()
     const {functions, usedCustomLayout: usedCustomLayoutForFunctionExtensions} = await this.loadFunctions(
```

Comment:
I think we'd expect to see `usedCustomLayout` changing if this config is used. If we add that behaviour then analytics are already in place to measure usage of this feature.